airflow/utils/dates.py

```diff
@@ -225,3 +226,14 @@ def days_ago(n, hour=0, minute=0, second=0, microsecond=0):
         second=second,
         microsecond=microsecond)
     return today - timedelta(days=n)
+
+
+def to_iso(execution_date):
+    """ Parse execution date to iso format. """
+    iso = execution_date
+    try:
+        iso = execution_date.isoformat()
+    except AttributeError:
```

Comment:
Yes execution_date can also be string format (any unicode representation instead of datetime) and this function is written to provide a universal way of getting iso format for execution_date instead of casing on it everytime. I can add more documentations for this function.